samples/nrf_desktop/src/hw_interface/motion_optical.c

```diff
@@ -744,13 +810,47 @@ static void optical_thread_fn(void)
 				cpi = new_cpi;
 				update_cpi(cpi);
 			}
+
```

Comment:
please hide this code behind some function

Reply:
done